superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx

```diff
@@ -160,10 +160,13 @@ afterAll(() => {
   fetchMock.resetBehavior();
 });
 
+const renderModal = (props: any) =>
```

Comment:
It would be a good idea to use the `PropertiesModalProps` type instead of `any`.

Reply:
One day we'll do a big "any" sweep again, but you're right, now's a good time to fix this one.

Reply:
Good point! Fixed